frontend/src/routes.jsx

```diff
@@ -17,7 +17,7 @@ let routes = (
     <DefaultRoute handler={Home} />
     <Route name='dashboard' handler={Dashboard} />
     <Route name='sprint' handler={SprintBoard} />
-    <Route name='project' handler={Project} />
+    <Route name='project' path='project/:id' handler={Project} />
```

Comment:
There is that router bug (i think), where if you transition to `/project/1` and refresh the page, an error occurs. I will try to look into this.